File: C/5.c
#include <stdio.h>

float somaImposto(float taxa, float custo);

int main(void)
{
  float taxa = 0;
  float custo = 0.0;
  printf("Digite a taxa do imposto(%): ");
  scanf("%f", &taxa);
  printf("Digite o preço/custo do produto: ");
  scanf("%f", &custo);

  float novo_valor = somaImposto(taxa, custo);
  printf("O novo valor é R$%.2f\n", novo_valor);
  return 0;
}
float somaImposto(float taxa, float custo)
{
  float novo_valor = ((taxa / 100) * custo) + custo;
  return novo_valor;
}

File: Python/avulsos/5.py
'''
Faça um programa com uma função chamada somaImposto. A função possui dois parâmetros 
formais: taxaImposto, que é a quantia de imposto sobre vendas expressa em porcentagem e 
custo, que é o custo de um item antes do imposto. A função “altera” o valor de custo para
incluir o imposto sobre vendas.
'''
def somaImposto(taxa, custo):
  novo_valor = ((taxa / 100) * custo) + custo
  return novo_valor

taxa = int(input("Digite a taxa de imposto sobre o produto(%): "))
custo = float(input("Digite o valor do produto: "))
print(f"O novo valor do produto é R${somaImposto(taxa, custo):.2f}")